src/custom/pages/Profile/LockedGnoVesting/index.tsx

```diff
@@ -114,16 +121,16 @@ const LockedGnoVesting: React.FC<Props> = ({ openModal, closeModal }: Props) =>
         <ConvertWrapper>
           <BalanceDisplay titleSize={18} altColor={true}>
             <i>
-              Vested{' '}
+              Claimable{' '}
               <MouseoverTooltipContent
                 wrap
                 content={
                   <div>
                     <p>
-                      <strong>COW vesting from the GNO lock</strong> is vested linearly over four years, starting on Fri
-                      Feb 11 2022 at 13:05:15 GMT.
+                      <strong>COW vesting from the GNO lock</strong> is vested linearly over four years, starting on{' '}
+                      {formatDateWithTimezone(new Date('02-11-2022 13:05:15 GMT'))}.
```

Comment:
May I suggest to had the date string and/or Date object in a constant rather than here?

Reply:
Added in here https://github.com/cowprotocol/cowswap/pull/460